.jenkins/job_beam_PostCommit_Java_RunnableOnService_Dataflow.groovy

```diff
@@ -26,7 +26,7 @@ mavenJob('beam_PostCommit_Java_RunnableOnService_Dataflow') {
   previousNames('beam_PostCommit_RunnableOnService_GoogleCloudDataflow')
 
   // Set common parameters.
-  common_job_properties.setTopLevelMainJobProperties(delegate)
+  common_job_properties.setTopLevelMainJobProperties(delegate, defaultTimeout:120)
```

Comment:
I only just learned it by googling. Perhaps we should have a linter on these files?